appium-cookies/lib/cookies.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+ //needed to communicate between JSON protocol cookies and regular js cookies 
+ // doesn't actually store the cookies kind of just translates them 
+import _ from 'underscore';
+import Helper from '../lib/helpers';
+export default class Cookies {
+  constructor (cookieString) {
+  this.ret = cookieString ? cookieString : undefined;
```

Comment:
Also, what is `ret`? Coming across it, I immediately think it is a return value, but then it doesn't make much sense in terms of a property of the class.